docs/document/docs/components/desktop/user-org-group-select.md

```diff
@@ -89,9 +89,9 @@ const userSelect = new Kuc.UserOrgGroupSelect({
   value: ['alice', 'marketing-group', 'acme-corp'],
   requiredIcon: true,
   error: 'Error occurred!',
-  className: 'options-class',
+  className: 'sample-class',
```

Comment:
I only meant to change component preview script.
https://github.com/kintone-labs/kintone-ui-component/pull/1619#discussion_r2245445022

```suggestion
  className: 'options-class',
```